.codacy/prospector.yaml

```
@@ -1,25 +1,24 @@
 output-format: json
-
-strictness: medium
+strictness: high
 test-warnings: false
 doc-warnings: true
 member-warnings: false
 uses:
- - django
+- django
```

YAMLlint (1.37.1)

[error] wrong indentation: expected at least 1

(indentation)